docs/pages/cheatsheet.adoc

```diff
@@ -351,15 +348,35 @@ Removed from service object.
 &#x5B; [purple]##**choices**## [purple]##**:**## &lt;expression> ] [purple]#**;**#
 ----
 
-=== Transfer function
+*Annotations:*
 
-Removed from service object.
+[options="header", cols="1,2"]
+|====================
+|Annotation | Meaning
+|@Lazy      | Defer fetching the object(s) as long as it is possible.
+|@Eager     | Fetch the object(s) as soon as it is possible.
+|====================
+
+=== Transfer action
+
+The action is a piece of code (currently Java implementation) that is executed when the action is invoked. An action may receive input as defined in its parameter.
+
+*Syntax:*
+
+[subs="quotes"]
+----
+[purple]#**action**# &#x5B;&lt;type>|&lt;union>|[purple]#**void**#] &lt;name> [purple]##**(**##&#x5B;&lt;parameter>][purple]##**)**## &#x5B;[purple]##**throws**## &lt;error1> &#x5B;, &lt;error2>] ...] [purple]#**;**#
+----
+
+=== Transfer submit
+
+The submit is a piece of code (currently Java implementation) that is executed when the submit is invoked. The submit receives the primitive fields and relations annotated with @Eager as its input. If there is no body defined for the submit, the default behavior is saving the transfer object's field to the mapped entity.
```

Comment:
Which CRUD operation is mapped for submit? Create? Update?

Reply:
How Submit can be distinguished to be used for Create or Update? What I have to do if I wanna disable Create function for Submit?

Reply:
If it is a must, we need @create and @update annotations for submit.

Reply:
Yes. It is have different schemathics in REST and in the frontend too.

Reply:
But it's still one method! It may update or create, or both.

Reply:
specified as it has been agreed